service/business/impl/getUserCompanyBranches.go

```diff
@@ -29,7 +29,7 @@ func (a businessService) GetUserCompanyBranches(w http.ResponseWriter, r *http.R
 		return errors.NewServerError(model.GetUserCompanyBranchesError, err.Error())
 	}
 
-	var companyBranches []model.CompanyBranch
+	var companyBranches = make([]model.CompanyBranch, 0)
```

Comment:
same with the above

Reply:
Ini menjawab response API biar tetep list in case hasil dari query masih kosong

Reply:
oke noted.